### PR TITLE
Feat: Support queues

### PR DESCRIPTION
**Pull request by lieldulev**
## Description
Add support to define a queue on a celery client (for placing a job or starting a worker).

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature: Add support to choose the queue in rabbitmq/redis

* **What is the current behavior?** (You can also link to an open issue here)
Currently the client always assumes the celery's queue name will be "celery", there is no way to choose a different one, and that value is hardcoded (in multiple places).

* **What is the new behavior (if this is a feature change)?**
Added an optional queue parameter in relevant classes.
"celery" is still the default queue, if none is passed on initialization. 

* **Other information**:
Also fixed multiple TS linting and updated the tslint rule for explicit any (since it is needed).

### CODE REVIEW

**Comment by actumn** [comment]
I really appreciate you made PR and implemented the feature !! 
It looks so great ! :)

But just a little thing, can you explain the reason you added dist folder on git?
I think we don't have to track dist folder.

**Comment by lieldulev** [comment]
Oh yes, you can remove it, or I can if you don't want it - There are reasons to save it in the repo and reasons not to. 

I've added because it simplifies point my app's package.json to my fork directly :) If you're planning on releasing changes frequently this is redundant.

**Comment by actumn** [comment]
Oh, I got it. 
We need dist folder just to publish on npm or module, and I guess it would be confusing if it is on the repository.
Can you remove it?

**Comment by actumn** [comment]
Thank you so much !!

File: .eslintrc [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "rules": {
     "no-console": 0,
     "@typescript-eslint/camelcase": 0,
+    "@typescript-eslint/no-explicit-any": 0,
```
Comment (erlichmen) [inline .eslintrc:21]: I wouldn't change the rules

File: src/app/worker.ts [hunk 2]
```diff
@@ -127,7 +127,7 @@ export default class Worker extends Base {
       }
 
       // request
-      const [args, kwargs, embed] = body;
+      const [args, kwargs /*, embed */] = body;
```
Comment (erlichmen) [inline src/app/worker.ts:130]: not required to change

File: src/app/worker.ts [hunk 3]
```diff
@@ -164,7 +164,7 @@ export default class Worker extends Base {
    * @todo implement here
    */
   // eslint-disable-next-line class-methods-use-this
-  public stop() {
+  public stop(): any {
```
Comment (erlichmen) [inline src/app/worker.ts:167]: not required to change

File: src/index.ts [hunk 4]
```diff
@@ -7,8 +7,8 @@ import Worker from "./app/worker";
  * @function
  * @returns {Client}
  */
-export function createClient(broker = "amqp://", backend = "amqp://"): Client {
-  return new Client(broker, backend);
+export function createClient(broker = "amqp://", backend = "amqp://", queue = "celery"): Client {
```
Comment (erlichmen) [inline src/index.ts:10]: put "celery" in a const

File: src/index.ts [hunk 5]
```diff
@@ -17,6 +17,6 @@ export function createClient(broker = "amqp://", backend = "amqp://"): Client {
  * @function
  * @returns {Worker}
  */
-export function createWorker(broker = "amqp://", backend = "amqp://"): Worker {
-  return new Worker(broker, backend);
+export function createWorker(broker = "amqp://", backend = "amqp://", queue = "celery"): Worker {
```
Comment (erlichmen) [inline src/index.ts:20]: put "celery" in a const

**Pull request merged**